tracktotrip/learn_trip.py

```diff
@@ -57,7 +57,7 @@ def learn_trip(current, current_id, canonical_trips, insert_canonical, update_ca
 
     if len(canonical_trips) == 0:
         current.simplify(eps, 0, 0, topology_only=True)
-        print("inserting trip %d" % len(current.points))
+        print(("inserting trip %d" % len(current.points)))
```

Comment:
Why are you changing prints so that they have double parenthesis?

Reply:
Automatically done by 2to3, I don't think it makes any difference, but can be done to have just one pair of parenthesis.